### ISSUE TITLE
Assign(or move window to workspace) electron apps switch you to another workspace

### ISSUE THREAD

**Issue by bugworm** [swaywm/sway#1213]
```
exec --no-startup-id discord
#for_window [class="discord"] move window to workspace 3
assign [class="discord"] 3
```
When main window shows up, desktop switch to workspace 3, it should only place window to that workspace. Same thing with atom, so it's probably an electron thing

**Comment by SirCmpwn** [comment]
https://github.com/SirCmpwn/sway/wiki#how-do-i-report-issues

**Comment by AnotherBugworm** [comment]
My bad
https://gist.github.com/AnotherBugworm/5a023e0e3614450f360da41d42d8c16e

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unreleased sway 1.0.
